assets/js/admin/admin-scripts.js

```diff
@@ -343,8 +340,24 @@ jQuery.noConflict();
 					5000
 				);
 			}
-		}
+		},
+
+		verify_settings: function () {
+			var success_setting = $('#success_page');
+			var failure_setting = $('#failure_page');
+
+			/**
+			 * Verify success and failure page.
+			 */
+			success_setting.add(failure_setting).change(function () {
+				if (success_setting.val() === failure_setting.val()) {
+					alert(give_vars.matched_success_failure_page);
```

Comment:
Rather than an alert can we display the warning on screen using a WP notice that is dismissible?